core/block_render_svg.js

```diff
@@ -77,10 +77,17 @@ Blockly.BlockSvg.CORNER_RADIUS = 8;
  */
 Blockly.BlockSvg.START_HAT = false;
 /**
+ * Height of the top hat.
+ * @const
+ */
+Blockly.BlockSvg.START_HAT_HEIGHT = 15;
+/**
  * Path of the top hat's curve.
  * @const
  */
-Blockly.BlockSvg.START_HAT_PATH = 'c 30,-15 70,-15 100,0';
+Blockly.BlockSvg.START_HAT_PATH = 'c 30,-' +
+  Blockly.BlockSvg.START_HAT_HEIGHT +' 70,-' +
```

Comment:
Indent twice more.  Space after +.
Don't worry about it, I'll fix this post-merge.